### DISCUSSION TITLE
堆叠和分组柱状图

### DISCUSSION THREAD

**Discussion by AnteVukusic** [ant-design/ant-design-charts#2778] [Q&A]
你好，我想问一下，有没有办法使用 Ant Design Charts 构建这种类型的图表？
![image](https://github.com/user-attachments/assets/56cf048a-ebfd-4f13-9e90-2ce6a5197aaf)

这是我的数据集：

savings_account_name - 柱状图的项目
date - X轴
amount - Y轴
total_amount - 柱状图的高度
saved_amount - 柱状图的高度
它应该按 date 分组，并且在每个日期上，每个 savings_account_name 都应该有一个柱状图，其中包含堆叠的 total_amount 和 saved_amount。

**Comment by lxfu1** [comment]
https://ant-design-charts.antgroup.com/examples/statistics/column/#group-and-stack
